packages/databases-collections/src/modules/drop-database/index.js

```diff
@@ -6,9 +6,6 @@ import isRunning, {
 import isVisible, {
   INITIAL_STATE as IS_VISIBLE_INITIAL_STATE
 } from '../is-visible';
-import name, {
-  INITIAL_STATE as NAME_INITIAL_STATE
-} from '../drop-database/name';
```

Comment:
This reducer looked unused, the name would be set once, but the `updateDatabaseName` event was never used.